docs/wiki/feature/mk6mortar.md

```diff
@@ -42,7 +42,7 @@ For this you need a `82mm Rangetable`, `Map Tools` and a `Vector 21` are also re
   - Under the `D ELEV for 100m DR` find the number that matches your ELEV and compensate:
      - _Example_, if you're 200m above your target multiply the number by 2 and add it to your ELEV.
      - If you're 200m below the target multiply the number by 2 and substract it to your ELEV.
-     - The lower the elevation the closer to you it will land.
+  - The higher the elevation the closer to you it will land.
```

Comment:
```suggestion
      - The higher the elevation the closer to you it will land.
```
The other two lines above are the context for this line, it should have the same indentation.